scanner/mapping_test.go

```diff
@@ -69,3 +69,29 @@ var _ = Describe("mapping", func() {
 		})
 	})
 })
+
+var _ = Describe("isSynced", func() {
+	lyrics1 := `NF Real Music
+	[00:00] ksdjjs
+	[00:00.85] JUST LIKE YOU
+	[00:00.85] Just in case my car goes off the highway`
+	lyrics2 := `[00:00] ksdjjs`
+	lyrics3 := "[04:02:50.85]"
+	lyrics4 := "[02:50.85]"
+	lyrics5 := "[02.50]"
+	lyrics6 := "[02:50]"
+	lyrics7 := "[]"
+	lyrics8 := ""
+	It("returns false if lyrics contain no timestamps", func() {
+		Expect(isSynced(lyrics5)).To(Equal(false))
```

Comment:
```go
Expect(isSynced("[02.50]")).To(Equal(false))
```
I don't see the point of rejecting timestamps that use `.` instead of `:`.
Is there an official format specification for subtitles?